addons/road-generator/ui/road_toolbar_create_menu.gd

```diff
@@ -29,40 +24,57 @@ enum CreateMenu {
 	LANE,
 	LANEAGENT,
 	TWO_X_TWO,
-	EXPORT_MESH
+	EXPORT_MESH,
+	LOCK_ROTATION_X,
+	LOCK_ROTATION_Y = 90,  # for some reason, this being `9` breaks the checkbox
```

Comment:
This is a _super_ weird bug that might be in the engine somewhere but I didn't have time to investigate.

By default, `LOCK_ROTATION_Y` here would be `9`.

For some reason, inside of `_toggle_check_item` below where it calls `get_item_index` with `LOCK_ROTATION_Y`, it would _always_ return an index of `9` (in reality the index is ~15).

This is only broken for the `enum` value of `9`. For all other values, it works fine... if you switch `LOCK_ROTATION_Y` and `LOCK_ROTATION_X` so that `LOCK_ROTATION_X` is `9`, then you get the same behavior there!

This is in `v4.5-rc2`